lib/src/sai_redis_notifications.cpp

```diff
@@ -12,6 +12,7 @@ sai_switch_shutdown_request_notification_fn on_switch_shutdown_request_notificat
 sai_fdb_event_notification_fn               on_fdb_event = NULL;
 sai_port_state_change_notification_fn       on_port_state_change = NULL;
 sai_packet_event_notification_fn            on_packet_event = NULL;
+sai_queue_pfc_deadlock_notification_fn      on_queue_deadlock = NULL;
```

Comment:
what if the real sai supports this notification natively?

Reply:
Orchagent subscribes no sairedis notification for this callback like to any other SAI notification. The difference is that if SAI supports native notifications, then it will trigger event, and if we have .lua plugins for the platform, then syncd will run them and trigger same callback itself.